homeassistant/components/media_player/mpd.py

```diff
@@ -141,7 +141,15 @@ def media_duration(self):
     @property
     def media_title(self):
         """ Title of current playing media. """
-        return self.currentsong['title']
+        name = self.currentsong['name']
```

Comment:
Is this key always available? Because if it is not, this will cause a `KeyError` to be raised.